composer/loggers/mlflow_logger.py

```diff
@@ -346,6 +346,55 @@ def log_model(self, flavor: Literal['transformers'], **kwargs):
             else:
                 raise NotImplementedError(f'flavor {flavor} not supported.')
 
+    def create_model_version(
+        self,
+        model_uri: str,
+        name: str,
+        await_creation_for: int = 300,
+        tags: Optional[Dict[str, Any]] = None,
+    ):
+        """Similar to MLflow's create_model_version, but will also create the initial model if it does not exist.
+
+        This function differs from ``register_model`` in that it also uses the run id.
```

Comment:
Why happens [here](https://github.com/mlflow/mlflow/blob/3ba1e50e90a38be19920cb9118593a43d7cfa90e/mlflow/tracking/_model_registry/fluent.py#L109)? Is that run_id not valid or is the mdoel_uri not a runs uri?

Reply:
Yeah because in our case the uri is a local path, there is not run to parse out of it.